packages/eslint/src/generators/workspace-rules-project/workspace-rules-project.ts

```diff
@@ -104,16 +107,6 @@ export async function lintWorkspaceRulesProjectGenerator(
   // Add swc dependencies
   addSwcRegisterDependencies(tree);
 
-  // Add extra config to the jest.config.ts file to allow ESLint 8 exports mapping to work with jest
-  addPropertyToJestConfig(
```

Comment:
Again, because we've always been generating this we will need a migration for existing workspaces to remove it